tests/formatters.txt

```diff
@@ -2,11 +2,11 @@
 # This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile --allow-unsafe --output-file=tests/formatters.txt --strip-extras tests/formatters.in
+#    pip-compile --allow-unsafe --cert=None --client-cert=None --index-url=None --output-file=tests/formatters.txt --pip-args=None --strip-extras tests/formatters.in
```

Comment:
@webknjaz Hi, could you take a look at this please? I'm somewhat stumped why `pip-compile` seems to be explicitly disabling some configuration.

Reply:
My guess is that a new release of pip-compile now puts all arguments into this command line, even the `None` ones. Whether this is a bug or a feature I don't know.

(The latest release of pip-tools is from July 2025, so I'm a bit curious why this didn't happen between then and now already...)

Reply:
I think this is actually being generated by `uv pip compile` now, so you'd want to look at the uv changelog, but it any case, the changes to this header indeed seem inconsequential.

Reply:
stable-2.18's noxfile still runs pip-compile. It installs it with `session.install("pip-tools >= 7", "pip < 24.3")`, so there's no version locking as for newer branches.

Reply:
So we either wait for a new pip-tools release (no idea how long that might take), or we restrict click to < 8.3.0.

Reply:
Hmm. I guess the constraints file that we're using applies to the docs-build packages so it seems like a hack to pin `click` there. Adding a new constraints file for the formatters and other checks seems like overkill.

I guess we can just merge these dep refreshes and ignore the broken check. It affects only stable-2.18 and stable-2.17 so we can keep an eye on it.

The next scheduled dep refresh probably won't fail this check again. We'd probably notice the pip-tools fix added when the check next fails. Hopefully that will remove the explicitly set config.

What do you think?

Reply:
I think I meant just for this specific case, as we're on the stable-2.18 branch. Although I take your point.

If we add a new constraints file it should first go on the `devel` branch. It seems wrong to me to introduce a constraints file in `stable-2.18`. However there is nothing to pin on `devel`. Should we just have an empty constrains file until we get to `stable-2.18`?